lib/src/better_feedback.dart

```diff
@@ -17,6 +17,7 @@ class BetterFeedback extends StatefulWidget {
     this.localizationsDelegates,
     this.localeOverride,
     this.mode = FeedbackMode.navigate,
+    this.pixelRatio = 3.0,
```

Comment:
Can you think of any other options/configurations? I wonder if it would make sense to create a `FeedbackConfig` in order to not have lot of parameters which just get passed down.

Though I currently can't think of any except the little wait duration.
It also needs an assert to make sure it's always greater than 0.

Reply:
I think this would make sense once we have multiple items that logically belong together. Navigation mode & pixelRatio for me doesn't, so I think it's easier to use as optional flag on its own - generally I prefer less overhead on syntax.